debian/zookeeper/Dockerfile.rpm

```diff
@@ -29,7 +29,11 @@ ENV COMPONENT=zookeeper
 
 RUN echo "===> installing ${COMPONENT}..." \
     && yum -q -y update \
-    && yum install -y confluent-kafka-${SCALA_VERSION}-${KAFKA_VERSION} \
+    && if [ "x${ALLOW_UNSIGNED}" = "xtrue" ]; then \
+        yum install -y --nogpgcheck confluent-kafka-${SCALA_VERSION}-${KAFKA_VERSION}${CONFLUENT_PLATFORM_LABEL}; \
```

Comment:
You can set `gpgcheck=1` in the `confluent.repo` file and it won't bother with anything gpg-related.